frontend/src/routes/Bookshelf.jsx

```diff
@@ -2,22 +2,23 @@ import '../styles/Bookshelf.css';
 
 import { useState, useEffect } from 'react';
 import Container from 'react-bootstrap/Container';
-import DocumentsCards from '../components/DocumentsCards';
+import GraphContainer from '../components/GraphContainer';
 
 function Bookshelf({backend, user}) {
+  console.log('Bookshelf');
   const [documents, setDocuments] = useState([]);
   const [lastUpdate, setLastUpdate] = useState();
 
   useEffect(() => {
+    console.log('Fetching documents...');
     backend.refreshDocuments(setDocuments);
   }, [lastUpdate, user, backend]);
-
   return (
-    <Container className="screen bookshelf">
-      <DocumentsCards docs={documents} byRow={4} createOn={[]}
-        {...{setLastUpdate, backend}}
-      />
-    </Container>
+    <>
```

Comment:
An empty elements is necessary in JSX only when the HTML fragment contains more than one root element.